### ISSUE TITLE
JS Error in locales: Uncaught SyntaxError: Unexpected reserved word

### ISSUE THREAD

**Issue by abolivar** [kartik-v/yii2-date-range#40]
```php
 'attribute' => 'fecha_vencimiento',
                    'hAlign' => 'center',
                    'vAlign' => 'middle',
                    'filterType' => GridView::FILTER_DATE_RANGE,
                    'filterWidgetOptions' => [
                    'language'=>'es',
...]
```
Uncaught SyntaxError: Unexpected reserved word in
import moment from '../moment';

**Comment by kartik-v** [comment]
The locale files have been copied from the plugin... can you check for any reserved words or BOM characters for your language and resubmit update via a PR?

**Comment by educationisenemy** [comment]
Hi. Have the same issue with russian locale.
I guess that problem in string `import moment from '../moment';`

**Comment by kartik-v** [comment]
Resolved via [upgrade to release v1.6.3](https://github.com/kartik-v/yii2-date-range/commit/fcfd163d5c1e2f299eb5f47e1aa487a86dfe56bd)
